src/svc_auth_gss.c

```diff
@@ -628,15 +653,22 @@ _svcauth_gss(struct svc_req *req, bool *no_dispatch)
 		 */
 		unref_svc_rpc_gss_data(gd);
```

Comment:
You should set gd_hashed to false here, in case code is added after this in the function.

Reply:
Done.